lib/scout_apm/utils/active_record_metric_name.rb

```diff
@@ -86,6 +93,8 @@ def parse_operation
       NON_GREEDY_CONSUME = '.*?'
       TABLE = '(?:"|`)?(.*?)(?:"|`)?\s'
       COUNT = 'COUNT\(.*?\)'
+      BEGIN_STATEMENT = 'BEGIN'.freeze # BEGIN is a reserved keyword
+      COMMIT = 'COMMIT'.freeze
```

Comment:
We should standardize the name of the constant between the two (`_STATEMENT` or not).

Also these constants make more sense down below with the `_LABEL` constants I think. 

Or we could just fold it directly into the code where it's used.